docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,116 @@
+tags:
+  - Release notes
+---
+
+# December 2023 release notes
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the repository so that it
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE Extensions. These changes
+  persist between restarts.
+* Added authentication provider information for each user under **Settings** >
+  **Access** > **Members** to help distinguish between users who log in with
+  multiple methods.
+* The metrics collected now include more granular information to help
+differentiate scans using different engine capabilities, such as intraprocedural
+scans without secrets validation versus intraprocedural scans *with* secrets
+validation.
+* **CLI tool**: Added new `semgrep test` subcommand, which is an alias for `semgrep scan
+--test`. **Note**: When running Semgrep on a test directory, use `semgrep
+scan test` to avoid confusion with the new `semgrep test` subcommand.
+
+### Changed
+
+* Improved the speed with which Semgrep Cloud Platform's **Dashboard** and **Findings**
+  pages load.
+* Fixed issue where empty tables in `pyproject.toml` files would fail to parse.
+* **OCaml**: Switched to a tree-sitter-based parser instead of the Menhir parser.
+* **Rust**: Updated the parser used for Rust.
+
+### Fixed
+
+* Fixed issue where webhooks stopped working.
+* Fixed issue so that clicking **Start Tour** now restarts the Getting Started
+  tutorial.
+* Fixed issue where the **Members** page doesn't display a user's new role until
+  the page reloads. <!--OS-1233-->
+* Fixed issue where users switching organizations would result in a 404.
+  <!--OS-136-->
+* Fixed the **Connect to** button under **Settings** > **Source Code Managers**
+  so that it displays correctly based on whether the user can connect to a
+  source code manager. <!-- https://github.com/semgrep/semgrep-app/pull/11812
+  -->
+* **CLI tool**: Updated CLI error message to clarify that users should log in before running
+  either:
+  * `semgrep ci`
+  * `semgrep scan` then `semgrep scan --config`
+  
+## 💻 Code
+
+### Fixed
+
+* Fixed issue where Semgrep Code findings marked as **fixed** can be triaged via the rule
+  group. If the first finding in the list of findings is fixed, the rest
```

Comment:
Not sure if this second sentence makes sense to me. I think just saying "if a rule is fixed, we can't change the triage status back to ignored" is enough?